client/src/app/pages/dashboard/maps/map-list/map-list.component.html

```diff
@@ -3,7 +3,8 @@
     <div class="col-12 col-sm-9 col-md-10">
       <div class="row align-items-center">
         <div class="form-col col-12 col-sm-10 col-md-6 col-lg-4 col-xl-3">
-          <input nbInput fullWidth class="form-control" type="text" placeholder="Map Name" formControlName="search">
+          <input nbInput fullWidth class="form-control" type="text" placeholder="Map Name" formControlName="search" onkeydown = "if (event.keyCode == 13)
+          document.getElementById('searchBtn').click()">
```

Comment:
Noting here, as mentioned on the call, this would probably be better if it were a form, where pressing enter submits the form (applies the search).

See [the guide on forms](https://angular.io/start/start-forms) for more info.

Reply:
Sounds good. I have discarded that change and added a form in its place - doing a little extra research to see if I can do something similar for the dropdown to keep the code clean.

Reply:
Update on the dropdown: while this solution works for now, I talked to TheNerdMan about potentially refactoring the button down the line to be more efficient with API usage and overall Angular principles using two-way bindings. If it's alright with you, I think it could be worth checking out as a new card enhancement; something I can tackle later as I learn how to tackle this problem